index.html

```diff
@@ -2,15 +2,73 @@
 <html lang="en">
 
 <head>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Itim&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
 </head>
 
+<section class="header-section">
+    <img class="logo" alt="a red-velvet cupcake logo" src="../images/Redvelvet_cupcake_logo.jpg">
```

Comment:
the page does not load for me locally because the path is `../` but I guess it is okay on netlify because it works there